### ISSUE TITLE
schedular

### ISSUE THREAD

**Issue by Trilokesh-Praxis-2023** [pytorch/pytorch#124367]
### 🐛 Describe the bug

/opt/conda/lib/python3.10/site-packages/torch/optim/lr_scheduler.py:149: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)

### Error logs

/opt/conda/lib/python3.10/site-packages/torch/optim/lr_scheduler.py:149: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)

### Versions

latest version

cc @vincentqb @jbschlosser @albanD @janeyx99 @crcrpar @ezyang @msaroufim @bdhirsh @anijain2305 @chauhang

**Comment by janeyx99** [comment]
Closing as duplicate to https://github.com/pytorch/pytorch/issues/103444. If you have another use case different from the one described in the issue above, please raise it there.
